pkg/controller/networkconfig/networkconfig_controller.go

```diff
@@ -0,0 +1,132 @@
+package networkconfig
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/pkg/errors"
+
+	networkoperatorv1 "github.com/openshift/cluster-network-operator/pkg/apis/networkoperator/v1"
+	"github.com/openshift/cluster-network-operator/pkg/apply"
+	"github.com/openshift/cluster-network-operator/pkg/network"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+)
+
+// Configuration objects must have a single name
+const CONFIG_OBJECT_NAME = "default"
+
+// The periodic resync interval.
+// We will re-run the reconciliation logic, even if the network configuration
+// hasn't changed.
+var ResyncPeriod = 5 * time.Minute
```

Comment:
I know this is the same as before, but I think @deads2k has told me that a resync that low is pointless...  @deads2k am I mis-remembering?  What number would you advise?